File: 01.마이크로서비스 아키텍처/02.구글 드라이브/googledrive/drive/src/main/java/google/drive/FileUploaded.java
package google.drive;


public class FileUploaded extends AbstractEvent {

    private Long id;
    private Long size;
    private String name;
    private Long userId;

    public FileUploaded(){
        super();
    }

    public Long getId() {
        return id;
    }

    public void setId(Long Id) {
        this.id = id;
    }
    public Long getSize() {
        return size;
    }

    public void setSize(Long Size) {
        this.size = size;
    }
    public String getName() {
        return name;
    }

    public void setName(String Name) {
        this.name = name;
    }
    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long UserId) {
        this.userId = userId;
    }
}

File: 01.마이크로서비스 설계 - MS Archi/02.구글 드라이브/googledrive_python/video processing/FileUploaded.py

from AbstractEvent import AbstractEvent
import json

class FileUploaded(AbstractEvent):
    id : int
    size : int
    name : str
    userId : int
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.size = None
        self.name = None
        self.userId = None

